lib/client/renderer.js

```diff
@@ -168,8 +173,11 @@ function init (client, d3) {
     function treatmentTooltip (d) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (10)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2417700254/issues/source?bid=2452379&fileBranchId=3233006#l173)